examples/oapp-solana/tasks/solana/skipNonce.ts

```diff
@@ -0,0 +1,48 @@
+import { publicKey, transactionBuilder } from '@metaplex-foundation/umi'
+import bs58 from 'bs58'
+import { task, types } from 'hardhat/config'
+import { ActionType, HardhatRuntimeEnvironment } from 'hardhat/types'
+
+import { normalizePeer } from '@layerzerolabs/devtools'
+import { types as devtoolsTypes } from '@layerzerolabs/devtools-evm-hardhat'
+import { EndpointId } from '@layerzerolabs/lz-definitions'
+
+import { myoapp } from '../../lib/client'
+
+import { deriveConnection, getExplorerTxLink, getLayerZeroScanLink, getSolanaDeployment } from '.'
+
+interface Args {
+    sender: string
+    fromEid: EndpointId
+    toEid: EndpointId
+    nonce: number
+}
+
+const action: ActionType<Args> = async (args, hre: HardhatRuntimeEnvironment) => {
+    const { fromEid, sender, nonce, toEid } = args
+    const solanaDeployment = getSolanaDeployment(toEid)
+    const counter: myoapp.MyOApp = new myoapp.MyOApp(publicKey(solanaDeployment.programId))
+    const { connection, umi, umiWalletSigner } = await deriveConnection(toEid)
+
+    const params: Pick<myoapp.instructions.SkipInboundNonceInstructionDataArgs, 'srcEid' | 'sender' | 'nonce'> = {
+        srcEid: fromEid,
+        sender: normalizePeer(sender, fromEid),
+        nonce: nonce,
+    }
+    const ixn = counter.skipInboundNonce(umiWalletSigner, params)
```

Comment:
The code attempts to use `skipInboundNonce` instruction in the OApp, but this instruction is not implemented in the program

Reply:
ah yeah, this file should be removed. thanks.